### PR TITLE
Fix date for adsense data test.

### PR DESCRIPTION
**Pull request by danielgent**
## Summary

Addresses issue #2585

## Checklist

- [ ] My code is tested and passes existing unit tests.
- [ ] My code has an appropriate set of unit tests which all pass.
- [ ] My code is backward-compatible with WordPress 4.7 and PHP 5.6.
- [ ] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [ ] My code has proper inline documentation.
- [ ] I have added a QA Brief on the issue linked above.
- [ ] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

### CODE REVIEW

File: .github/workflows/visual-regression.yml [hunk 1]
```diff
@@ -12,6 +12,7 @@ on:
       - '!assets/**/__tests__/**/*.js'
       - '!assets/**/test/*.js'
       - '!assets/**/*.test.js'
+      - 'stories/**'
```
Comment (aaemnnosttv) [inline .github/workflows/visual-regression.yml:15]: Let's also add a pattern for matching stories written in CSF which may not be in this directory.

**Review by aaemnnosttv — COMMENTED** [review]
LGTM – just one more thing to add.

File: .github/workflows/visual-regression.yml (hunk 1 above)
Comment (danielgent) [reply .github/workflows/visual-regression.yml]: Makes sense @aaemnnosttv. I changed the pattern and you can see the VRTs are running!

Comment (aaemnnosttv) [reply .github/workflows/visual-regression.yml]: Thanks @danielgent, although the fact that they're running now can also be attributed to this path 😉 
https://github.com/google/site-kit-wp/blob/890e24fac0144fdcc08f4eb02c54c4170f34e757/.github/workflows/visual-regression.yml#L10

The new pattern should match them all though so +1 to this change 👍

**Review by aaemnnosttv — APPROVED** [review]
Great, thanks!

**Pull request merged**